swagger.yaml

```diff
@@ -56,24 +67,52 @@ paths:
         200:
           description: OK
           schema:
-            $ref: "#/definitions/SearchResponse"
+            $ref: "#/definitions/DepartmentResponse"
         400:
           description: Query term not specified
         500:
           description: Internal server error
 
-  /departments/search:
+  /health:
     get:
       tags:
-        - department
-      summary: "Department query API"
-      description: "Department query API which returns matching links to other departments"
+        - health
+      summary: "Health check API"
+      description: "API to query the app for current health status"
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Health"
+        500:
+          description: Internal Server Error
+
+  /search:
+    get:
+      tags:
+        - search
+      summary: "Default ONS query API"
+      description: "Default ONS query API which combines the content with type counts"
       parameters:
         - in: query
           name: q
-          description: "Query search term"
+          description: "Query search term."
           type: string
           required: true
+        - in: query
+          name: content_type
+          description: "Comma separated list of content types to be returned."
```

Comment:
An example or list of available content types might be useful

Reply:
I haven't actually touched this code. It's github misunderstanding what was on develop and what is on this PR. You are right but it is out of scope of this ticket. I've raised the issue and we will continue updating the search endpoint at a later date.

Reply:
Yes this is github not understanding the code changes correctly and has miss matched the changed lines - Noelle has not changed the search endpoint doc so out of scope - fyi - we will be changing logic on this endpoint going forwards so we will be revisiting so I wouldn't get too hung up on this now